src/commands/base-command.ts

```diff
@@ -192,11 +192,6 @@ export default class BaseCommand extends Command {
       .addOption(new Option('--cwd <cwd>').hideHelp(true))
       .addOption(new Option('-o, --offline').hideHelp(true))
       .addOption(new Option('--auth <token>', 'Netlify auth token').hideHelp(true))
-      .addOption(
-        new Option('--httpProxy [address]', 'Old, prefer --http-proxy. Proxy server address to route requests through.')
-          .default(process.env.HTTP_PROXY || process.env.HTTPS_PROXY)
-          .hideHelp(true),
-      )
```

Comment:
It looks like there was an `httpProxy` option and an `http-proxy` option but these got accidentally(?) squashed into a duplicated `httpProxy` option in https://github.com/netlify/cli/pull/5816/files#diff-67082469659283bc342bd13f7e20570cb3d1bf5096aad88545a9ac4809d6083bL106-L140. The latest version of commander throws when you declare duplicate options.

What's extra confusing is the copy here implies that `http-proxy` was intended to be the _new_ one but that's not where it landed 🤷🏼.

Reply:
Handled in https://github.com/netlify/cli/pull/7046